engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/AdditionalDictComponentConfigsExtractor.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.management.periodic
```

Comment:
I'm not sure if this is the right place. It's sth related to DeploymentService (mostly), but it's also used in PeriodicProcessService. Do you see some common place for these two things?

Reply:
It was quite problematic to put it into a right place as PeriodicProcessService is defined in `nussknacker-flink-periodic-manager` and DeploymentService is defined in `nussknacker-designer`. I decided to move it to util package of deployment-manager-api